docs/recipes/code-coverage.md

```diff
@@ -54,7 +54,13 @@ First, we need a Babel configuration. The following is just an example. You will
 ```json
 {
 	"babel": {
-		"presets": ["es2015"],
+		"presets": [
+	    ["env", {
```

Comment:
Babel seems to be [documenting these](https://www.npmjs.com/package/@babel/preset-env) as `@babel/preset-env` rather than just `env`.